doc/styleguide.md

```diff
@@ -0,0 +1,248 @@
+# Node.js Documentation Style Guide
+
+These are the guidelines for writing Node.js documentation.
+
+## Headings
+
+* Each page must have a single `#`-level title at the top.
+* Chapters in the same page must have `##`-level headings.
+* Sub-chapters need to increase the number of `#` in the heading according to
+  their nesting depth.
+* The page's title must follow [APA title case][title-case].
+* All chapters must follow [APA sentence case][sentence-case].
+
+Using a hypothetical `Quick Start` as an example:
+
+```markdown
+# How to write a test for the Node.js project
```

Comment:
This is sentence case but above it indicates this must be title case.

Reply:
> This is sentence case but above it indicates this must be title case.

```suggestion
# How to Write a Test for the Node.js Project
```